skiko/src/jsMain/kotlin/org/jetbrains/skiko/Actuals.js.kt

```diff
@@ -1,8 +1,25 @@
 package org.jetbrains.skiko
 
-import org.w3c.dom.events.MouseEvent
+import org.w3c.dom.HTMLElement
 import org.w3c.dom.events.UIEvent
 
-internal actual fun getEventTimestamp(e: UIEvent): Long {
+internal fun getEventTimestamp(e: UIEvent): Long {
     return e.timeStamp.toLong()
-}
\ No newline at end of file
+}
+
+internal actual fun CursorManager_setCursor(component: Any, cursor: Cursor) {
+    if (component is HTMLElement) {
```

Comment:
It's just an issue of taste and I don't have any strong view about this, I'm just say that the other option would be to just 

```
internal actual fun CursorManager_setCursor(component: Any, cursor: Cursor) {
    (component as? HTMLElement)?.style?.cursor = cursor
}

internal actual fun CursorManager_getCursor(component: Any): Cursor? {
    return (component as? HTMLElement)?.style?.cursor
}
```

again, I'm not even sure this has any benefits, just wanted to discuss

Reply:
fixed.

The benefit is that it's a shorter version :) but still readable.